09_maps/main.go

```diff
@@ -69,22 +69,39 @@ func main() {
 	fmt.Println(human)
 
 	// TODO: Print the length of the map human
+	fmt.Println(len(human))
 
 	// TODO: Check if the key "age" exists in the map human
 	// If so print "age exists in the map"
 	// If not print "age does not exist in the map"
 	// Do the same for the key "weight"
```

Comment:
You missed this one

Reply:
What do you mean? It seems to print, `2`, isn't that the maps length?

Reply:
I meant this `// Do the same for the key "weight"`

Reply:
Done.